scalastyle-test-config.xml

```diff
@@ -0,0 +1,99 @@
+<scalastyle>
```

Comment:
Lot's of duplicated XML and afaik scalastyle doesn't support `include` directive. Could you add a comment at the top of the file describing differtences between this file and `scalastyle-config.xml` ?